extended/src/main/java/io/kubernetes/client/extended/generic/options/DeleteOptions.java

```diff
@@ -0,0 +1,5 @@
+package io.kubernetes.client.extended.generic.options;
+
+import io.kubernetes.client.openapi.models.V1DeleteOptions;
+
+public class DeleteOptions extends V1DeleteOptions {}
```

Comment:
what is the purpose of this class?

Reply:
i added a class named `ListOptions` in this package and it's probably good to gather those option class into the same package. the `DeleteOptions` here is just a link to the generated delete option class. wdyt?